ci/properties/grunt.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Grunt",
+    "description": "Build a Grunt project with npm.",
+    "iconName": "nodejs",
```

Comment:
Better to have icon for gulp/grunt etc instead of nodejs?

Reply:
yes, updated
This is how these will look on recommending - 
![image](https://user-images.githubusercontent.com/23234879/126264106-ea1fe443-bf94-4039-a2f0-151c0ddc4f7d.png)